nf_core/modules/info.py

```diff
@@ -93,7 +93,8 @@ def init_mod_name(self, module):
                 if self.repo_type == "modules":
                     modules = self.get_modules_clone_modules()
                 else:
-                    modules = self.modules_json.get_all_modules().get(self.modules_repo.fullname)
+                    modules = self.modules_json.get_all_modules().get(self.modules_repo.remote_url)
+                    modules = [module if dir == "nf-core" else f"{dir}/{module}" for dir, module in modules]
```

Comment:
Reading quickly so can't see the full context, but why do we need special treatment for `nf-core` here?

Reply:
This is to get the module name suggestions, it will usually be nf-core, so we don't need to have `nf-core/TOOL`, but in case you have modules from other repos I thought it would be nice to differentiate and have the other directory listed.